### PR TITLE
Fix cobra 98

### PR DESCRIPTION
**Pull request by logicalangel**

### CODE REVIEW

File: internal/config/config.go [hunk 1]
```diff
@@ -26,13 +28,12 @@ func Load(configPath, secretPath string) error {
 	SecretFilePath = secretPath
 	err := cleanenv.ReadConfig(secretPath, &App.Secret)
 	if err != nil {
-		log.Logger.With("Error", err).Warn("Can't read secret file")
+		pureLog.Println("Can't load secrets")
```
Comment (pouya-eghbali) [inline internal/config/config.go:31]: why

File: internal/config/model.go [hunk 2]
```diff
@@ -4,9 +4,15 @@ import (
 	"time"
 )
 
+// System struct hold the internal system configuration.
 type System struct {
 	Name string `env:"SYSTEM_NAME" env-default:"Unchained" yaml:"name"`
-	Log  string `env:"SYSTEM_LOG"  env-default:"info"      yaml:"log"`
+	Log  string `env:"SYSTEM_LOG"  env-default:"debug"     yaml:"log"`
```
Comment (pouya-eghbali) [inline internal/config/model.go:10]: default should be info

File: internal/crypto/ethereum/rpc.go [hunk 3]
```diff
@@ -5,9 +5,10 @@ import (
 	"fmt"
 	"sync"
 
+	contracts2 "github.com/KenshiTech/unchained/internal/crypto/ethereum/contracts"
```
Comment (pouya-eghbali) [inline internal/crypto/ethereum/rpc.go:8]: contracts2?

File: internal/crypto/identity.go [hunk 4]
```diff
@@ -0,0 +1,137 @@
+package crypto
```
Comment (pouya-eghbali) [inline internal/crypto/identity.go:1]: this should be moved to internal/crypto/secp256k1/identity.go

File: internal/service/uniswap/uniswap.go [hunk 5]
```diff
@@ -177,46 +179,46 @@ func (u *Service) RecordSignature(
 		return
 	}
 
-	if isMajority {
-		reportLog := log.Logger.
-			With("Block", info.Asset.Block).
-			With("Price", info.Price.String()).
-			With("Token", info.Asset.Token.Name)
+	if !isMajority {
```
Comment (pouya-eghbali) [inline internal/service/uniswap/uniswap.go:182]: Check the file;

```go
	DebouncedSaveSignatures(
		info.Asset,
		SaveSignatureArgs{Hash: hash, Info: info},
	)
```

and 

```go
	if !isMajority {
		log.Logger.Debug("Not a majority")
	}
```

are duplicated. Are there any other duplicates in other files??

**Review by pouya-eghbali — CHANGES_REQUESTED** [review]

File: internal/config/config.go (hunk 1 above)
Comment (logicalangel) [reply internal/config/config.go:31]: You know that we read the logger level from the configurations. These codes are loaded before initializing the logger. Before this PR, the logger's level which is passed to the logger was empty.

**Review by pouya-eghbali — APPROVED** [review]

**Pull request merged**